doc/architecture/decisions/003-authoring-schema-variants.md

```diff
@@ -0,0 +1,41 @@
+# 3. Generate Schema variants from a data source + manifest
+
+Date: 04/07/2017
+
+## Status
+
+Proposed
```

Comment:
I think we agreed that the status of "Proposed" is implied by it being in PR so maybe we should set this to "Accepted" as it will be once merged? we also missed this with the previous decisions